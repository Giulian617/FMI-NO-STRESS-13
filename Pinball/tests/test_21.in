100 11717
74 55 9
51 74 4
28 55 10
62 28 4
40 51 8
23 74 9
32 51 6
2 23 4
26 51 4
86 23 9
47 23 5
87 26 2
20 23 3
14 55 6
11 20 9
37 32 3
78 2 10
31 87 1
49 23 6
39 49 5
65 37 3
46 37 7
89 62 10
90 62 8
61 40 7
33 86 6
25 39 5
10 39 4
17 25 9
81 87 10
93 2 7
1 20 1
52 33 9
19 1 5
67 81 1
21 33 7
75 32 8
12 47 7
82 67 1
69 74 10
54 10 2
27 17 3
64 2 1
68 11 10
7 74 1
42 61 10
99 40 9
53 52 8
50 7 6
41 32 8
98 89 5
43 10 7
63 2 10
48 31 1
59 39 4
100 40 2
66 68 8
9 23 5
45 51 2
71 7 3
58 54 5
57 93 5
29 40 10
79 40 1
24 98 1
18 54 2
60 27 9
15 12 5
94 32 3
38 33 7
76 46 1
83 52 5
92 90 8
16 38 2
8 39 10
4 82 6
88 4 7
5 82 2
97 10 7
84 90 8
36 37 5
91 20 10
96 11 3
77 82 8
85 98 1
6 87 8
35 47 5
13 4 5
56 27 5
30 47 2
3 82 9
22 67 5
80 21 4
73 10 8
70 33 5
34 25 3
95 56 4
72 87 1
44 2 7
