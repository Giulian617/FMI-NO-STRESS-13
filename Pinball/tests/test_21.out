882950776
373591237
524808501
116847454
7942273
631522287
749909619
526495301
741891375
626348466
375278449
919601496
731010408
61527561
301096401
107694154
527050143
647868624
526817486
725459658
965684100
832895790
207171273
778656451
655057402
846447692
832895790
686956457
986158385
631573070
801418472
559964540
895401555
541289516
440663844
624483338
126467169
778656451
258265226
526817486
914771655
486888234
801418472
348419412
741891375
586401350
395751568
731010408
31367811
504729697
395751568
644088734
346356685
677047359
965684100
757585201
588200927
